lmn/templates/lmn/users/user_profile.html

```diff
@@ -1,19 +1,18 @@
 {% extends 'lmn/base.html' %}
 {% block content %}
 
-
-<!-- A user's profile page.
-
-Includes list of user's notes with title and preview of text.
-Text truncated to 200 characters. 
-
-user_profile is the user that this profile is about 
-user is a variable provided to the template and is the current logged-in user 
-
--->
-
-<h2 id='username_notes'>{{ user_profile.username }}'s notes</h2>
-
+<h2 id='username_profile'>{{ user_profile.username }}'s Profile</h2>
+<p>edit profile here</p>
+
+<form method="POST" action="{% url 'my_user_profile' %}">
```

Comment:
There's already a form in my_user_profile.html for a user to edit their profile. This of this  template as a read-only way for anyone to view anyone else's profile page, and my_user_profile as the current logged in user's profile page.